modules/index.js

```diff
@@ -0,0 +1,94 @@
+import Book from './book.js';
+import UI from './ui.js';
+import Store from './store.js';
+import { DateTime } from './luxon.js';
+
+/* Navigation selectors */
+const listNav = document.querySelector('.list');
+const addNav = document.querySelector('.new-book');
+const contactNav = document.querySelector('.contact');
+
+/* Pages selectors */
+const listPage = document.querySelector('#list');
+const addPage = document.querySelector('#book-form');
+const contactPage = document.querySelector('#contact');
+
+// time selector
+const displayTime = document.getElementById('date');
+
+function refreshTime() {
+  const dateString = DateTime.now().toRFC2822();
+  displayTime.textContent = dateString;
+}
```

Comment:
- [ ] It will be great to convert this function into an arrow function. Remember we are converting everything into `es6` syntax